examples/demo-rollup/src/rollup.rs

```diff
@@ -82,6 +86,43 @@ pub async fn new_rollup_with_celestia_da(
     })
 }
 
+/// Creates MockDa based rollup.
+pub fn new_rollup_with_mock_da(
```

Comment:
Should we add a runtime flag to `demo-rollup/main` so we can choose between celesta and mock da?
If yes then:
1. Which one would should be the default?
2. We will need to update the readme accordingly.

Reply:
Hmm, yeah. In general, I think we want using different da layers to be a compile time change - but for the demo maybe it's nice to be able enable mock da at runtime. Maybe we should save this for a follow up?

Reply:
Chatted with @bkolad offline - we'll save this for a follow-up